src/Components/Aspire.Azure.Messaging.EventHubs/EventHubsComponent.cs

```diff
@@ -89,30 +90,52 @@ protected static void EnsureConnectionStringOrNamespaceProvided(AzureMessagingEv
                 $"'ConnectionStrings:{connectionName}' or specify a 'ConnectionString' or 'FullyQualifiedNamespace' in the '{configurationSectionName}' configuration section.");
         }
 
-        // If we have a connection string, ensure there's an EntityPath if settings.EventHubName is missing
+        // Emulator: If we have a connection string, ensure there's an EntityPath if settings.EventHubName is missing
```

Comment:
```suggestion
        // If we have a connection string, ensure there's an EntityPath if settings.EventHubName is missing
```

This isn't Emulator specific (although the emulator does go down this path. You can use this library with a connection string to a real azure service.